From e357ab82b6309a7ffffdb87a55fb4973861e4962 Mon Sep 17 00:00:00 2001
From: Avinash Kumar <avikumar@habana.ai>
Date: Wed, 30 Jun 2021 08:22:18 +0300
Subject: [PATCH] [SW-0] preboot: calling uart flush function

Call to preboot_uart_rbuf_flush needed for
previous uart log "Waiting for commands.."
to be visible during i2c debug mode when
preboot is waiting for host gpio.

Change-Id: I0bb9561b5d71e81688dd6de6f67a44edd2917bee
Signed-off-by: Avinash Kumar <avikumar@habana.ai>
---

diff --git a/common/cmd_module.c b/common/cmd_module.c
index 484f22c..a81db46 100644
--- a/common/cmd_module.c
+++ b/common/cmd_module.c
@@ -39,6 +39,7 @@
 		cmd_ops[j].setup();
 
 	preboot_puts("Waiting for commands...\n");
+	preboot_uart_rbuf_flush();
 
 	i = 0;
 	while (1) {
